hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/TransitRegionStateProcedure.java

```diff
@@ -372,14 +372,12 @@ private Flow confirmOpened(MasterProcedureEnv env, RegionStateNode regionNode)
   }
 
   private void closeRegionAfterUpdatingMeta(MasterProcedureEnv env, RegionStateNode regionNode) {
-    CloseRegionProcedure closeProc =
+    LOG.debug("Close region: isSplit: {}: evictOnClose: {}", isSplit,
       isSplit
```

Comment:
Nit: this log is misleading, we should just pass the `evictCache` valure for `evictOnClose`.